src/db/models/index.js

```diff
@@ -1,15 +1,12 @@
-const connection = require('../connection');
```

Comment:
So this part was probably the most problematic for me. A model importing the connection module from higher up the hierarchy. Two issues with this: it's like a generic `Button` component importing a `Main` class. It's a bit anti-pattern. It's also very highly susceptible to encountering circular dependency issues: https://en.wikipedia.org/wiki/Circular_dependency.